src/java.base/share/classes/javax/crypto/CipherSpi.java

```diff
@@ -298,10 +298,11 @@ protected abstract void engineSetPadding(String padding)
     protected abstract AlgorithmParameters engineGetParameters();
 
     /**
-     * Initializes this cipher with a key and a source
+     * Initializes this {@code Cipher} object with a key and a source
```

Comment:
Hmm, with this {@code } change, I think the class name should be `CipherSpi` instead of `Cipher`.
This comment applies to the rest of this file.

Reply:
Fixed. That's a lot of changes.